app/test_split_integer.py

```diff
@@ -2,20 +2,29 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(8, 2)
+    assert result[0] == result[1]
```

Comment:
It's ok, but lets rewrite in one line and remove variable to make it more self-explanatory